charts/kspm-collector/README.md

```diff
@@ -45,11 +45,13 @@ The following table lists the configurable parameters of the Sysdig KSPM Collect
 | `clusterName`                              | Sets a unique cluster name. This name will be used to identify events using the `kubernetes.cluster.name` tag.                                    | ` `                                                         |
 | `image.registry`                           | Specifies the KSPM collector image registry.                                                                                                      | `quay.io`                                                   |
 | `image.repository`                         | Specifies the image repository to pull from.                                                                                                      | `sysdig/kspm-collector`                                     |
-| `image.tag`                                | Specifies the image tag to pull from the image repository.                                                                                        | `1.39.11`                                                   |
+| `image.tag`                                | Specifies the image tag to pull from the image repository.                                                                                        | `1.39.12`                                                   |
 | `image.digest`                             | Specifies the image digest to pull from the image repository.                                                                                     | ` `                                                         |
 | `image.pullPolicy`                         | Specifies theImage pull policy.                                                                                                                   | `""`                                                        |
 | `imagePullSecrets`                         | Specifies the Image pull secret.                                                                                                                  | `[]`                                                        |
 | `replicas`                                 | Specifies the KSPM collector deployment replicas.                                                                                                 | `1`                                                         |
+| `transportLayer`                                 |  Determine the transport used by the collector to receive tasks from the backend and send results back to it. Valid values are "nats" and "http".                                                                                              | `http`                                                         |
```

Comment:
this seems not to be formated right?

Reply:
it seems ok https://github.com/sysdiglabs/charts/blob/ma/kspm-collector/bump-to-v1.39.12/ssprod-57042/charts/kspm-collector/README.md